src/main.ts

```diff
@@ -169,3 +216,25 @@ app.on('activate', () => {
     createWindow()
   }
 })
+
+if (!app.isDefaultProtocolClient(pkg.protocolSchemes.native)) {
+  // Define custom protocol handler. Deep linking works on packaged versions of the application!
+  app.setAsDefaultProtocolClient(pkg.protocolSchemes.native)
+}
+
+app.on('will-finish-launching', function () {
+  // Protocol handler for osx
+  app.on('open-url', function (event, url) {
+    event.preventDefault()
+    deeplinkingUrl = url
+    if (mainWindow) {
+      redirectDeepLinkingUrl(deeplinkingUrl.toString(), mainWindow)
+    }
+  })
+})
+
+ipcMain.on('app-ready', () => {
+  if (mainWindow && deeplinkingUrl) {
+    redirectDeepLinkingUrl(deeplinkingUrl.toString(), mainWindow)
+  }
+})
```

Comment:
Same as above. I think handlers for `ipcMain.on` and `app.on` events can go to the feature folder and be tested

Reply:
Updated and created test